src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/core/utils/PsiTypeUtils.java

```diff
@@ -154,7 +161,7 @@ public static boolean isBinary(PsiModifierListOwner psiMember) {
     }
 
     public static boolean isOptional(String fieldTypeName) {
-        return fieldTypeName.startsWith("java.util.Optional");
+        return fieldTypeName.startsWith("java.util.Optional<");
```

Comment:
This method is used for another code, I think it should be better to have a new method like `isOptionalWithTypeParameter`